runtime/src/stash_measurement.rs

```diff
@@ -4,65 +4,77 @@ use crate::{dpe_crypto::DpeCrypto, CptraDpeTypes, DpePlatform, Drivers};
 use caliptra_common::mailbox_api::{
     MailboxResp, MailboxRespHeader, StashMeasurementReq, StashMeasurementResp,
 };
-use caliptra_drivers::{CaliptraError, CaliptraResult};
+use caliptra_drivers::{pcr_log::PCR_ID_STASH_MEASUREMENT, CaliptraError, CaliptraResult};
 use crypto::{AlgLen, Crypto};
 use dpe::{
     commands::{CommandExecution, DeriveChildCmd, DeriveChildFlags},
     context::ContextHandle,
     dpe_instance::DpeEnv,
     response::DpeErrorCode,
 };
-use zerocopy::FromBytes;
+use zerocopy::{AsBytes, FromBytes};
 
 pub struct StashMeasurementCmd;
 impl StashMeasurementCmd {
     pub(crate) fn execute(drivers: &mut Drivers, cmd_args: &[u8]) -> CaliptraResult<MailboxResp> {
         if let Some(cmd) = StashMeasurementReq::read_from(cmd_args) {
-            let hashed_rt_pub_key = drivers.compute_rt_alias_sn()?;
-            let pdata = drivers.persistent_data.get();
-            let rt_pub_key = pdata.fht.rt_dice_pub_key;
-            let mut crypto = DpeCrypto::new(
-                &mut drivers.sha384,
-                &mut drivers.trng,
-                &mut drivers.ecc384,
-                &mut drivers.hmac384,
-                &mut drivers.key_vault,
-                rt_pub_key,
-            );
-            let mut env = DpeEnv::<CptraDpeTypes> {
-                crypto,
-                platform: DpePlatform::new(
-                    pdata.manifest1.header.pl0_pauser,
-                    hashed_rt_pub_key,
-                    &mut drivers.cert_chain,
-                ),
+            let dpe_result = {
+                let hashed_rt_pub_key = drivers.compute_rt_alias_sn()?;
```

Comment:
Since we're using the `?` operator here, I would expect `dpe_result` to be a `CaliptraResult`, but it seems like we don't have to return `Ok(error_code)` at the end of this block. So i'm a little confused about what happens to any errors from this funciton.

Reply:
I tested this out by making compute_rt_alias_sn return an error, and it is working as expected - the stash measurement mailbox command exits the compute_rt_alias_sn and returns the same error.